CONTRIBUTING.md

```diff
@@ -79,4 +114,45 @@ ensuring they pass
 
 ## Releasing
 
-TODO: Define release workflow once we have a PyPI publishing in place
+To create a tag and release, follow the instructions in this section.
+
+### Update the version, changelog, and notices
+
+1. Create a new branch for the version bump.
+
+1. Based on the unreleased content, determine the new version number and update the version in `version.py`
+
+1. Review the git log and ensure the changelog contains all relevant recent changes with references to GitHub issues or PRs, if possible.
+
+1. Review the changes since the last tag, and if the dependencies have changed revise the notices file to correctly capture the added dependencies and their licenses / copyrights.
```

Comment:
can we link to the notices file?

Reply:
we don't have a notices file 😱 . I will add that as a TODO but in the meantime link the NOTICES file so it will be valid